src/TreeMap/App.svelte

```diff
@@ -138,15 +139,15 @@
                 <div class="treeCounterComponent">
                     <div class="treeCounter">
                         <div class="textContainer">
-                            <p class="treecount">
+                            <p class={`treecount ${theme === 'dark' ? "planted" : ""}`}>
                                 {community
```

Comment:
This is still not working for me using community=false - community is always true if set as a property.